README.md

```diff
@@ -1,2 +1,18 @@
 # terraform-aws-cloudfront-bucket
-Terraform module creating an S3 bucket with a CloudFront distribution in front
+
+This modules creates an S3 bucket with a CloudFront distribution in front.
```

Comment:
I think it would make sense to explain that the integration between CloudFront and the S3 bucket is protected and it's not possible to access the bucket directly... which is what makes this a bit complicated.